### PR TITLE
feat: support trace sofa-rpc

### PR DESCRIPTION
**Pull request by gxcsoccer**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: lib/carrier/rpc_carrier.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+class RPCCarrier {
+
+  inject(spanContext) {
+    const header = {};
+    header['x-b3-traceId'] = spanContext.traceId;
```
Comment (popomore) [inline lib/carrier/rpc_carrier.js:7]: sofa-rpc 也是用这几个头？

**Review by popomore — APPROVED** [review]
LGTM

File: lib/carrier/rpc_carrier.js (hunk 1 above)
Comment (gxcsoccer) [reply lib/carrier/rpc_carrier.js:7]: 我没看 sofa-rpc 的，暂时就保留了

Comment (popomore) [reply lib/carrier/rpc_carrier.js:7]: https://github.com/alipay/sofa-rpc/blob/master/extension-impl/extension-common/src/main/java/com/alipay/sofa/rpc/common/RemotingConstants.java#L98

**Comment by gxcsoccer** [comment]
1.1.0

**Pull request merged**